applications/nrf_desktop/doc/dfu_mcumgr.rst

```diff
@@ -69,6 +69,8 @@ Make sure that the DFU lock utility is enabled if your nRF Desktop application c
 You cannot use this module with the :ref:`caf_ble_smp`.
 In other words, you cannot simultaneously enable the :ref:`CONFIG_DESKTOP_DFU_MCUMGR_ENABLE <config_desktop_app_options>` option and the :kconfig:option:`CONFIG_CAF_BLE_SMP` Kconfig option.
 
+Currently, this module supports only one bootloader backend: the MCUboot bootloader backend.
```

Comment:
```suggestion
Currently, this module supports only one bootloader backend, the MCUboot bootloader backend.
```

Reply:
Consider combining this sentence with the note below to highlight it (this is the same information):

```

.. note::
    Currently, this module supports only one bootloader backend, the MCUboot bootloader backend.
    B0 bootloader is not integrated with MCUmgr.
```

Reply:
This sentence is an introduction to the subsequent section about the MCUboot backend. I would then duplicate this information.